apps/api/src/app/inbox/usecases/snooze-notification/snooze-notification.command.ts

```diff
@@ -0,0 +1,13 @@
+import { IsDate, IsDefined, IsMongoId } from 'class-validator';
+import { Type } from 'class-transformer';
+import { EnvironmentWithSubscriber } from '../../../shared/commands/project.command';
+
+export class SnoozeNotificationCommand extends EnvironmentWithSubscriber {
+  @IsDefined()
+  @IsMongoId()
+  readonly notificationId: string;
+
+  @Type(() => Date)
+  @IsDate()
```

Comment:
```suggestion
  @IsDefined()
  @Type(() => Date)
  @IsDate()
```

Reply:
The `@IsDate()` on inwards DTO will handle if its undefined from user perspective calling controller, when using this usecase/command internally, you will get static and compile error when you don't provide a Date, so it should be fine.